src/pages/signatures/index.mdx

```diff
@@ -1,12 +1,12 @@
 ---
 product: signatures
 title: Criipto Signatures
-subtitle: Sign documents and PDFs with national eIDs and the PAdES standard
+subtitle: Sign PDF documents with national eIDs and the PAdES standard
```

Comment:
we also partly allow XML documents, perhaps that was the reason for the old phrasing, not sure :D

Reply:
Ah right! Should we keep it?

Reply:
@nmoskaleva Perhaps just "Sign documents"